testutils/infra/device.py

```diff
@@ -272,6 +275,13 @@ def ssh_is_opened(self, wait=60 * 60):
         for dev in self._devices:
             dev.ssh_is_opened(wait)
 
+    def get_client_service_name(self):
+        # We assume that the service name is always the same across all devices,
+        # so it's enough to return the first one.
+        assert len(self._devices) > 0
+        for dev in self._devices:
+            return dev.get_client_service_name()
```

Comment:
This is fine, but for simplicity I would have used `return self._devices[0].get_client_service_name()`

Reply:
Yes, you're right, somehow I got the idea that this was a map. Will fix.